test-it/testList.json

```diff
@@ -31,7 +31,8 @@
         },
         "manual": {
             "tests": [
-                "StringFormatTime.via"
+                "StringFormatTime.via",
+                "TestVI_LargeBDTest.via"
```

Comment:
Running "TestVI_LargeDBTest.via" on the framework causes Padding.via to fail on the second run. Padding.via fails with "Cannot enlarge memory arrays". I did some debugging and it appears that TestVI_LargeBDTest.via is not leaking. So something else is going on: memory fragmentation? Skipping for now.